app/service-accounts/server/methods/getLoginToken.js

```diff
@@ -0,0 +1,29 @@
+import { Meteor } from 'meteor/meteor';
+import { Accounts } from 'meteor/accounts-base';
+
+import { Users } from '../../../models';
+
+Meteor.methods({
+	getLoginToken(username) {
```

Comment:
What about the case when the user corresponding to neither `Meteor.userId` and nor `username` has a `.u` property?

Reply:
Ohh yes, I need to throw an error for that.

Reply:
Due to high security concerns in this method, I'd recommend structuring it like..  for the case we want to return, generate and return.. for everything else raise Error